__tests__/__unit__/Profiles.unit.test.ts

```diff
@@ -281,6 +284,27 @@ describe("Profile class unit tests", () => {
             (profiles.loadNamedProfile as any).mockReset();
           });
 
+        it("should rePrompt credentials", async () => {
+        const promptProfile = {name: "profile1", profile: {user: "oldfake", password: "oldfake"}};
+        profiles.loadNamedProfile = jest.fn(() => {
+            return promptProfile as any;
+        });
+
+        Object.defineProperty(ZosmfSession, "createBasicZosmfSession", {
+            value: jest.fn(() => {
+                return { ISession: {user: "fake", password: "fake", base64EncodedAuth: "fake"} };
+            })
+        });
+
+        showInputBox.mockResolvedValueOnce("fake");
+        showInputBox.mockResolvedValueOnce("fake");
+        const res = await profiles.promptCredentials(promptProfile.name, true);
+        expect(res[0]).toBe("fake");
```

Comment:
I would change it to 1 call instead of 3:
`expect(res).toEqual(["fake", "fake", "fake"])`
It's also more readable.